_posts/2023-03-09-multilayer-maps.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title:  "Getting started with multilayer maps in OpenSearch"
+authors:
+  - vijay
+  - navneev
+  - junqiu
+  - satnandi
+  - kolchfa
+date:   2023-03-09
+categories:
+  - technical-post
+meta_keywords: 
+meta_description: 
+excerpt: OpenSearch 2.5 introduced new multilayer maps for visualizing geographical data and narrating your story. Displaying multiple layers on a map lets you show data from different sources at different zoom levels. Additionally, in OpenSearch 2.6, you can add multilayer maps to dashboard panels within OpenSearch Dashboards, which makes it easier to analyze your geospatial data in the context of other visualizations. 
+---
+
+[OpenSearch 2.5](https://opensearch.org/docs/2.5/dashboards/maps-plugin/) introduced new multilayer maps for visualizing geographical data and narrating your story. Displaying multiple layers on a map lets you show data from different sources at different zoom levels. Let’s say you’re building a real estate application. Suppose you have an index with the number of houses available at the county level, another index with the number of houses available at the city level, and the third index with detailed information about individual houses. Initially, you may want to show the user a map with county boundaries and data. As the user zooms in, you’ll show city boundaries and data. As the user zooms in even more, you’ll display each house with information like the price, number of bedrooms, and so on. With the new multilayer maps, you can build various layers data from different data sources to accomplish this task.
+
+Multilayer maps let you:
+
+* Add multiple layers to visualize your data.
+* View or analyze geographical data that comes from different data sources.
+* View different data at different zoom levels.
+* Visualize additional data with tooltips.
+* Continuously refresh data for a real-time dataset.
```

Comment:
```suggestion
* Continuously refresh data, yielding a real-time dataset.
```

Reply:
Leaving as is because the meaning is not the same.